pkg/schema/annotations.go

```diff
@@ -297,6 +358,15 @@ func collectDocumentationAnnotations(node yamlmeta.Node) ([]Annotation, error) {
 			anns = append(anns, ann)
 		}
 	}
+	for _, annotation := range []template.AnnotationName{AnnotationTitle} {
+		ann, err := processOptionalAnnotation(node, annotation, nil)
+		if err != nil {
+			return nil, err
+		}
+		if ann != nil {
+			anns = append(anns, ann)
+		}
+	}
```

Comment:
We've got duplicate code, here. The `for` loop is iterating over the annotation names.

I think you meant to add `AnnotationTitle` to the list of `AnnotationName`s.

Reply:
Good catch. I have updated the code.